exercises/D-logical-operators/dist/exercise2.dev.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+/*
+  Logical Operators
+  ---------------------------------
+  This program calls some functions that are either missing or incomplete.
+  Update the code so that you get the expected result.
+*/
+function isNegative(num) {
+  if (num < 0) {
+    return true;
+  } else if (num > 0) {
+    return false;
+  } else {
+    return 0;
+  }
+}
```

Comment:
Great try. You really consider negative, zero and positive inputs. However, as seen from test cases in the end of the file, this function should only return either true or false. Zero is normally considered as a non-negative number.